packages/core/contracts/common/implementation/MultiCaller.sol

```diff
@@ -1,7 +1,9 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
```

Comment:
In the OA we have `solidity 0.8.16`, is it voluntary to this version here?

Reply:
we are actually a bit inconsistant in the repo with this...I think most of common is like this.